README.md

```diff
@@ -49,6 +49,9 @@ If you want `ember server` to start the Hoodie Server for you, you have to confi
 // config/environment.js
 if (environment === 'development') {
   ENV.hoodie = {
+    client: {
+      url: 'http://127.0.0.1:4201'
+    },
```

Comment:
I think the url is not necessary, because ember is proxying all request from `http://127.0.0.1:4200/hoodie/*` to `http://127.0.0.1:4201/hoodie/*` – can you check please?